js/ui/components/codingGameService.js

```diff
@@ -24,6 +24,33 @@ const CodingChatboxTextService = new Lang.Class({
         }
     },
 
+    _updateListeningForEvents: function() {
+        if (!this._service) {
+            return;
+        }
+
+        // In some cases, this can be null, just ignore it if so
+        if (!this._service.currently_listening_for_events) {
+            return;
+        }
+
+        let currentlyListeningFor = this._service.currently_listening_for_events.deep_unpack().map(s => s[0]);
```

Comment:
I think this will need to be updated when you follow my suggestion to use `as` as the property type